deploy/certmanager/070-ValidatingWebhookConfiguration.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: strimzi-drain-cleaner
+  labels:
+    app: strimzi-drain-cleaner
+  annotations:
+    cert-manager.io/inject-ca-from: strimzi-drain-cleaner/strimzi-drain-cleaner
+webhooks:
+  - name: strimzi-drain-cleaner.strimzi.io
+    rules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        operations:  ["CREATE"]
+        resources:   ["pods/eviction"]
+        scope:       "Namespaced"
+    clientConfig:
+      service:
+        namespace: "strimzi-drain-cleaner"
+        name: "strimzi-drain-cleaner"
+        path: /drainer
+        port: 443
+      caBundle: Cg==
```

Comment:
What's this `Cg==` value here as CA bundle?

Reply:
This is an empty string in base64. Cert Manager updates it after the deployment.

Reply:
Right!